### DISCUSSION TITLE
Outil de recherche d'acteurs

### DISCUSSION THREAD

**Discussion by Synfactory** [witsa/synapps#801] [Ideas]
Salut, ce serait top de pouvoir filtrer les recherches d'acteurs dans les déroulants en tapant son nom de clé 👍 
![image](https://github.com/user-attachments/assets/771827bf-5334-4ce8-a7cf-594c4d549680)

Un truc du genre : 
![image](https://github.com/user-attachments/assets/9b96b394-c1df-4349-bbf6-3be8b23b64bb)

**Comment by Raphwit** [comment]
On note, merci du retour !
